### PR TITLE
chore: bump deps, reduce log level, resume process

### PR DESCRIPTION
**Pull request by Zxilly**
#### What type of PR is this?

chore, feat

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [x] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### More detailed description for this PR(en: English/zh: Chinese).

bump delve to support go1.23 better, reduce log level as it didn't stop the process, resume the target process after the analysis done.

#### Which issue(s) this PR fixes:

Updates: #17

### CODE REVIEW

File: .github/workflows/release-check.yml [hunk 1]
```diff
@@ -20,6 +20,6 @@ jobs:
           SOURCE_VERSION=`grep 'Version\s*=\s*\"v[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}\"' *.go | awk -F '\"' '{print $(NF-1)}'`
           git checkout main
           MASTER_VERSION=`grep 'Version\s*=\s*\"v[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}\"' *.go | awk -F '\"' '{print $(NF-1)}'`
-          git checkout ${{Head.SHA}}
+          git checkout ${{ github.sha }}
```
Comment (GuangmingLuo) [inline .github/workflows/release-check.yml:23]: What's the reason to change this ?

Comment (Zxilly) [reply .github/workflows/release-check.yml:23]: https://github.com/cloudwego/goref/actions/runs/10297485050
image

Comment (Zxilly) [reply .github/workflows/release-check.yml:23]: Also, I think this workflow needs update, as python2 will be got removed in the github actions runner soon.

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md

image

Comment (GuangmingLuo) [reply .github/workflows/release-check.yml:23]: Indeed, all the release action failed before. Thanks for helping to fix it. Please help to fix the python scripts.

Comment (Zxilly) [reply .github/workflows/release-check.yml:23]: The error was not introduced by this PR.

Comment (GuangmingLuo) [reply .github/workflows/release-check.yml:23]: I think we can remove the release-check.yaml file in this repo. Cause we don't actually have a release procedure for this repo. All PRs are direactly merged to main branch. cc @jayantxie

Comment (jayantxie) [reply .github/workflows/release-check.yml:23]: LGTM, please just remove .github/workflows/release-check.yml. @Zxilly

**Review by jayantxie — APPROVED** [review]

**Pull request merged**